ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ClaimMapper.java

```diff
@@ -8,6 +8,7 @@
 import uk.gov.hmcts.cmc.domain.models.party.Party;
 
 import java.util.List;
+import java.util.Objects;
```

Comment:
can you look at the coverage report in sonar to see what's still missing please?
CNP was preventing me from deploying because it said coverage was too low :(

Reply:
Looks like only ` src/main/java/uk/gov/hmcts/cmc/claimstore` is configured for coverage, none other packages are configured.